tools/explorer/pkg/stellar/stellar.go

```diff
@@ -597,6 +599,13 @@ func (w *Wallet) GetNetworkPassPhrase() string {
 	}
 }
 
+// getMinumumBalance calculates minimum balance for an escrow account
+// following formula is used: minimum Balance = (2 + # of entries) * base reserve
+// entries is the amount of operations are required to setup the account
+func (w *Wallet) getMinumumBalance() int {
+	return (2 + len(w.assets) + len(w.signers)) * (1e7 / 2)
```

Comment:
lets make this `1e7` a constant (and set it to 10000000 written out so it for sure is never a float), so it is immediatly clear what this is supposed to mean